### ISSUE TITLE
Running own project

### ISSUE THREAD

**Issue by avinsrid** [sdnhub/SDNHub_Opendaylight_Tutorial#6]
Hello,

Most tutorials have mentioned on how we can create an OSGI project and start it using ODL.

For eg, on this link: http://www.frank-durr.de/ I can see that we first build using mvn to obtain the jar file. Then we go into the controller's distribution directory and ./run.sh . Once the controller is started we are starting the process of our application jar using start <pid> on the OSGI> console.

I noticed that in the SDNHub_Opendaylight_Tutorial, we are starting the controller from this directory itself. Could you kindly explain how we can do something like this instead of manually starting the process from the osgi console always?

Basically, I am interested in having an application for controller where I can start it myself through ./run.sh ?

Thanks and I appreciate the help!

~Avinash
